addons/rose/addon/styles/rose/components/table/_table.scss

```diff
@@ -35,7 +39,33 @@
     }
   }
 
+  // Grow to full available width
+  .rose-table-cell-expand {
```

Comment:
I was thinking the main use case is actually for a "shrink" concept rather than expand.